core/include/kernel/abort.h

```diff
@@ -27,11 +27,15 @@ struct abort_info {
 
 /* Print abort info to the console */
 void abort_print(struct abort_info *ai);
+
 /* Print abort info + stack dump to the console */
 void abort_print_error(struct abort_info *ai);
 
 void abort_handler(uint32_t abort_type, struct thread_abort_regs *regs);
 
+/* Platform overload, should be implemented in platform code */
+void plat_abort_handler(struct thread_abort_regs *regs);
```

Comment:
Isn't it only external aborts we care about here? If so, how about `plat_external_abort_handler()`?
This function should probably take a `struct abort_info *ai` argument instead.

Reply:
Yes, it is about external aborts. Maybe we can even restrain the call to platform code on external aborts that are not table walks or table updates?
I'm fine with your suggestions